web-ui/src/components/member-directory/MemberSummaryCard.jsx

```diff
@@ -51,9 +53,9 @@ const MemberSummaryCard = ({ member, index }) => {
         {isAdmin && (
           <div className="member-card-actions">
             <CardActions>
-              <Button onClick={handleOpen}>Edit Member</Button>
-              <Button>Terminate Member</Button>
-              <Button>Delete Member</Button>
+              <Button onClick={handleOpen}>Edit</Button>
+              <Button>Terminate</Button>
+              <Button>Delete</Button>
```

Comment:
Let's convert these to a split button: https://material-ui.com/components/button-group/#split-button

(can make another story)

Reply:
I'll make a note for story planning